### ISSUE TITLE
Odd phrasing - 'last poem' topic

### ISSUE THREAD

**Issue by desertdessert** [Monika-After-Story/MonikaModDev#9156]
In the `monika_lastpoem` topic, Monika refers to the act 2 Save Me poem as "the last poem I showed you". But this isn't actually the _last_ poem if the player has seen poems added by MAS (e.g. Valentine's day poems). This makes it sound like Monika is not aware of these additional poems when she brings up this topic. Maybe it could be reworded to something more generic, like 'that weird poem I showed you'?

**Comment by Justformas** [comment]
"the last poem I showed you" is officially how it's worded in DDLC though. Yes this is a topic straight from Act 3, as those are things Monika says. In the next line she explained that she's referring to the one right before Yuri killed herself.

However I will say that even if we aren't counting MAS, that's technically not the last poem from Monika in DDLC, because she shows you the Happy Ending poem in Act 3. But at the time in Act 3, her saying "the last poem I showed you" might make more sense than later in MAS. But it's still clear what she's referring to in the next line, and this is coming from someone who tends to interpret things multiple ways.
